Yank/tests/test_yaml.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+
+#=============================================================================================
+# MODULE DOCSTRING
+#=============================================================================================
+
+"""
+Test YAML functions.
+
+"""
+
+#=============================================================================================
+# GLOBAL IMPORTS
+#=============================================================================================
+
+import os
+import tempfile
+import textwrap
+
+from yank.yamlbuild import YamlBuilder
+
+#=============================================================================================
+# UNIT TESTS
+#=============================================================================================
+
+def test_yaml_parse():
+    """Check that YAML file is parsed correctly."""
+
+    yaml_text = """
+    ---
+    options:
+        wrong_option: 3
+        timestep: 1.0 * femtoseconds
+        nsteps_per_iteration: 2500
+    """
+
+    yaml_file = tempfile.NamedTemporaryFile(delete=False)
+    try:
+        # Check that handles no options
+        yaml_file.write('---\ntest: 2')
```

Comment:
What about rewriting this as
```
yaml_contents = """\
---
test: 2
"""
yaml_file.write(yaml_contents)
```
so that we can later easily add more to the string for testing parsing?